addons/beehave/nodes/beehave_tree.gd

```diff
@@ -93,3 +100,22 @@ func enable() -> void:
 
 func disable() -> void:
 	self.enabled = false
+
+
+func _enter_tree() -> void:
+	# Get the name of the parent node name for metric
+	var parent_name = get_parent().name
```

Comment:
Using `get_parent()` here is wrong, as there are different ways to associate a node to a tree (see `actor_node_path` export property)
```suggestion
	var parent_name = actor.name
```